src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -41,6 +42,72 @@ def persist_flow_run(self, run_info: FlowRunInfo):
         self.queue.put(run_info)
 
 
+class LineProcess:
+    def __init__(self, executor_creation_func):
```

Comment:
Maybe `LineProcessManager` is more suitable, since it actually holds the process initialization function and processes, not only a process object.